incubator-dolphinscheduler-website.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
what's this file used for?

Reply:
Thanks for your review.
incubator-dolphinscheduler-website.iml is auto create by IDEA . we project have no .gitignore file . so I well close this pull request and add a .gitinnore file .